app/src/main/cpp/skyline/input/sections/TouchScreen.h

```diff
@@ -6,13 +6,29 @@
 #include "common.h"
 
 namespace skyline::input {
+    static constexpr std::size_t MaxTouchPoints{16};
+
+    /**
+     * @brief Indicates if touch point has started or ended
+     * @url https://switchbrew.org/wiki/HID_services#TouchAttribute
+     */
+    union TouchAttribute {
+        u32 raw{};
+        struct {
+            bool start : 1;
+            bool end : 1;
+        };
+    };
+    static_assert(sizeof(TouchAttribute) == 0x4);
+
     /**
      * @brief A descriptor for a single point on the touch screen
      * @url https://switchbrew.org/wiki/HID_Shared_Memory#TouchScreenStateData
      */
     struct TouchScreenStateData {
         u64 timestamp; //!< The timestamp in samples
-        u32 _pad0_;
+
+        TouchAttribute attribute; //!< The attribute of this thouch
```

Comment:
touch* but I will add that this is somewhat older code and we have stopped doing comments like this where reading the code would imply the same thing faster so you can just entirely remove this comment really.